### PR TITLE
Filter out test and deleted Stripe customers from tmp view

### PR DESCRIPTION
**Pull request by jeffsloan**
Unsure if we need to go back and cast booleans as strings, so that these filters are warehouse-agnostic (e.g. [here in the macro for customer columns](https://github.com/fivetran/dbt_stripe_source/blob/0cfc3b4b91fede6d3f03d587da54e0272215ccc1/macros/get_customer_columns.sql#L24))

This works in Snowflake, though.

### CODE REVIEW

File: models/tmp/stg_stripe__customer_tmp.sql [hunk 1]
```diff
@@ -1,2 +1,4 @@
 select * 
 from {{ var('customer') }}
+where livemode = TRUE
```
Comment (fivetran-kristin) [inline models/tmp/stg_stripe__customer_tmp.sql:3]:
```suggestion
where livemode
```

File: models/tmp/stg_stripe__customer_tmp.sql [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+  and is_deleted = FALSE
```
Comment (fivetran-kristin) [inline models/tmp/stg_stripe__customer_tmp.sql:4]:
```suggestion
  and not is_deleted
```

**Review by fivetran-kristin — COMMENTED** [review]
Left two suggestions, which should make this compatible across the different warehouses.  LMK if this works on your side.

**Comment by jeffsloan** [comment]
Made the changes! Works on my side 👍

**Comment by fivetran-joemarkiewicz** [comment]
@jeffsloan wanted to respond here that I will be taking a look at this PR this week and will update you if I have any questions prior to merging and releasing the new stripe source version. Thanks!

**Comment by jeffsloan** [comment]
Sounds good! Any q's let me know

**Comment by fivetran-joemarkiewicz** [comment]
@jeffsloan there were a few cosmetic changes I wanted to make before merging with master and also want to test cross db compatibility with CircleCi. Therefore, I updated the base branch to the new `jsloan-updates` branch.  

I will approve and merge to this branch. We can continue any need conversation there then merge to master from a new PR.

**Pull request merged**